app/src/main/res/layout/main_activity.xml

```diff
@@ -6,6 +6,7 @@
               android:orientation="vertical">
 
     <android.support.v7.widget.Toolbar
+        android:id="@+id/toolbar"
```

Comment:
could we change all ids in this file to match the `main_activity_` prefix pattern?